lib/user/getPlayerInfo.js

```diff
@@ -18,33 +17,48 @@ exports.required = ['userId']
 **/
 
 // Define
-exports.func = function (args) {
-  return http({
-    url: '//www.roblox.com/users/' + args.userId + '/profile',
-    options: {
-      resolveWithFullResponse: true,
-      followRedirect: false
-    }
-  })
-    .then(function (res) {
-      if (res.statusCode === 200) {
-        const body = parser.load(res.body)
+function getPlayerInfo (userId) {
+  return new Promise((resolve, reject) => {
+    const requests = [
+      http({
+        url: '//www.roblox.com/users/' + userId + '/profile',
+        options: {
+          resolveWithFullResponse: true,
+          followRedirect: false
+        }
+      }),
+      http({
+        url: '//users.roblox.com/v1/users/' + userId,
+        options: {
+          resolveWithFullResponse: true,
+          followRedirect: false,
+          json: true
+        }
+      })
+    ]
+
+    Promise.all(requests).then(function (responses) {
+      const profileResponse = responses[0]
+      const usersResponse = responses[1]
+
+      if (profileResponse.statusCode === 200 && usersResponse.statusCode === 200) {
```

Comment:
This can't resolve banned users.

`profileResponse` has status code 302 when a banned user is called, thus causing the method to throw error 'User does not exist'.

Use the Roblox ID, [`8166491`](https://users.roblox.com/v1/users/8166491) to test this.